.github/workflows/report.yml

```
@@ -24,7 +24,7 @@ jobs:
         uses: exasol/python-toolbox/.github/actions/python-environment@0.14.0
```

suggestion: could update these to use newer versions, but not strictly necessary
```suggestion
        uses: exasol/python-toolbox/.github/actions/python-environment@0.20.0
```

---

Yes, why not?